be/src/exec/pipeline/exchange/sink_buffer.cpp

```diff
@@ -194,12 +194,39 @@ int64_t SinkBuffer::_network_time() {
 void SinkBuffer::cancel_one_sinker(RuntimeState* const state) {
     if (--_num_uncancelled_sinkers == 0) {
         _is_finishing = true;
+        if (state != nullptr && state->query_ctx() && state->query_ctx()->is_query_expired()) {
+            // how many in-flight rpcs and what exchange receivers are.
+            if (_total_in_flight_rpc > 0) {
+                std::stringstream ss;
+                auto remain_rpc_num = 0;
+                for (auto& remain_rpc : _num_in_flight_rpcs) {
+                    std::lock_guard<Mutex> l(*_mutexes[remain_rpc.first]);
+                    if (remain_rpc.second > 0) {
+                        ss << (remain_rpc_num > 0 ? ", " : "") << print_id(_instance_id2finst_id[remain_rpc.first]);
+                        remain_rpc_num++;
+                    }
+                }
+                LOG(WARNING) << "Fragment " << print_id(_fragment_ctx->fragment_instance_id()) << " SinkBuffer remains "
```

Comment:
can we use profile counter or info string  instead of LOGto  some succinct and crucial info?

Reply:
some cases may disable profile, finally depends on logs.